pages/about.html

```diff
@@ -1,12 +1,41 @@
-<!DOCTYPE html>
+<!DOCTYPE html> 
 <html lang="en">
 <head>
-  <meta charset="UTF-8">
+  <meta charset="UTF-8"> 
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>About</title>
+  <link href="../styles/style.css" rel="stylesheet"/>
 </head>
 <body>
-  
+  <header>
+    <h1 class="page-title">About Alie </h1>
```

Comment:
Since you only have a single `h1` per page the `page-title` class is redundant here:

```suggestion
    <h1>About Alie</h1>
```